index.js

```diff
@@ -28,6 +28,9 @@ module.exports = class Trailpack {
     if (!pack.config) {
       pack.config = { }
     }
+    if (!pack.config.trailpack) {
```

Comment:
I think we need :) in case user fill the config with wrong values we need to check config is fill with correct types or throw an error